lib/elasticpress-integration.php

```diff
@@ -10,6 +10,9 @@ function bootstrap() {
 	if ( ! defined( 'ELASTICSEARCH_HOST' ) ) {
 		return;
 	}
+	if ( defined( 'HM_ENV_TYPE' ) && HM_ENV_TYPE === 'local' ) {
```

Comment:
I wonder if we can just assume this is now defined, given the addition ot load.php